freedom@FREEDOMs-MacBook-Pro project-ml-microservice-kubernetes % ./run_kubernetes.sh      
Error from server (AlreadyExists): pods "udacitymicro" already exists
NAME           READY   STATUS    RESTARTS   AGE
udacitymicro   1/1     Running   0          8m39s
Forwarding from 127.0.0.1:8000 -> 80
Forwarding from [::1]:8000 -> 80
^C%                                                                                                                                                                        freedom@FREEDOMs-MacBook-Pro project-ml-microservice-kubernetes % kubectl logs udacitymicro
 * Serving Flask app "app" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: on
 * Running on http://0.0.0.0:80/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 231-779-307
[2020-05-11 21:07:19,589] INFO in app: JSON payload: 
{'CHAS': {'0': 0}, 'RM': {'0': 6.575}, 'TAX': {'0': 296.0}, 'PTRATIO': {'0': 15.3}, 'B': {'0': 396.9}, 'LSTAT': {'0': 4.98}}
[2020-05-11 21:07:19,600] INFO in app: Inference payload DataFrame: 
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2020-05-11 21:07:19,608] INFO in app: Scaling Payload: 
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2020-05-11 21:07:19,612] INFO in app: Output prediction: 
[20.35373177134412]
127.0.0.1 - - [11/May/2020 21:07:19] "POST /predict HTTP/1.1" 200 -
freedom@FREEDOMs-MacBook-Pro project-ml-microservice-kubernetes % 
